wordpress-mobile/WordPress-iOS
# Blaze: Add entry point for dashboard

Part of #20090

- Adds an entry point for the dashboard menu item
- Button actions will be addressed in a different PR
- Analytics will be addressed in a different PR
- Figma: cEGtlsm6xijEswaKkrQrIj-fi-477%3A5675

## How to test

0. Make sure that Blaze is enabled via the debug menu
1. Switch to a site that's eligible for Blaze (See peeHDf-zm-p2 for more info on Blaze requirements)
2. Switch to the dashboard
3. ✅ A Blaze dashboard card is displayed
4. Switch to a site that's not eligible for Blaze
5. ✅ A Blaze dashboard card is NOT displayed

<img src="https://user-images.githubusercontent.com/6711616/220160874-a3d59666-e3cc-4b1f-b7b8-74129a4d1fdc.png" width=200>

## Regression Notes
7. Potential unintended areas of impact
n/a

8. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

9. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
